README.md

```diff
@@ -41,217 +41,1885 @@ The required view creation scripts are found in:
 
 ```sql
 SELECT
-    id, machine_name, name, value, description, created_at
+    id, automation_system_name, config_key, config_value, description, created_at
 FROM
-    `biosero_test`.`configurations`
+    `biosero_uat`.`configurations`
 WHERE
-    machine_name = 'Workcell 1'
+    machine_name = 'CPA'
```

Comment:
What does CPA stand for?

Reply:
Cherrypick workcell A
As opposed to the Beckmans which are called CP1 to 4
A bit limited but RnD decided these names